packages/integration-examples/vite-react-typescript/src/fonts/font-face.scss

```diff
@@ -0,0 +1,80 @@
+// Path to font files in the css-library dist folder
+$css-library-font-path: "../../node_modules/@department-of-veterans-affairs/css-library/dist/fonts";
+
+/* latin - Source Sans Pro Regular */
+@font-face {
```

Comment:
It might be possible for a team to leverage a [USWDS font-face mixin](https://github.com/uswds/uswds/blob/e22700f3de57c799d39ff4a950b6b72a70b8cf6c/packages/uswds-core/src/styles/mixins/general/font-face.scss) to generate these quicker. I wasn't able to get it to work though.